tox.ini

```diff
@@ -44,6 +44,8 @@ deps =
     cov: pytest-cov
     oldestdeps: minimum_dependencies
     rad: rad @ git+https://github.com/spacetelescope/rad.git@main#egg=rad
+    devdeps: astropy>=0.0.dev0
+    devdeps: numpy>=0.0.dev0
```

Comment:
This is doing nothing, `pip` will just fall back on the latest version in the PyPi index (i.e. the latest release in it). If you want to use the "pre-built" presumably nightly binaries, you will have to specify extra indexes for pip to use.

https://github.com/spacetelescope/jdaviz/blob/0194237dcc45a5cda6d837d39b6527edada71d47/tox.ini#L72
indicates a possibility for astropy, though it should build just fine off main

https://github.com/astropy/astropy/blob/337798f36c9aa2ae98190c5819bde758d3aa5a40/tox.ini#L27 is how you specify the extra index for numpy nightly wheels.